src/sentry/replays/blueprints/api.md

```diff
@@ -91,6 +93,7 @@ Retrieve a collection of replays.
 | finished_at       | string                        | The **latest** timestamp received.                     |
 | id                | string                        | The ID of the Replay instance.                         |
 | is_archived       | bool                          | Whether the replay was deleted or not.                 |
+| misery_score      | number                        | The quality of the user's experience.                  |
```

Comment:
Lets get this term vetted, since this can potentially cause confusion due to being defined in a different scope: https://docs.sentry.io/product/performance/metrics/#user-misery

Reply:
I'll remove it for now and focus on the core rage/dead click attributes.  We can discuss naming and necessity of score fields later.